python/pyspark/sql/tests/test_connect_basic.py

```diff
@@ -79,6 +79,11 @@ def test_simple_explain_string(self):
         result = df.explain()
         self.assertGreater(len(result), 0)
 
+    def test_select_start(self):
+        df = self.connect.read.table(self.tbl_name).select("*")
+        data = df.toPandas()
+        self.assertEqual(len(data.index), 100)
```

Comment:
The best is to support `dataframe.schema` then check the schema. However the StructType support in proto is under development: https://github.com/apache/spark/pull/38200, and there could be other hidden issues there. 

Once schema is supported in proto, I will follow up to use `.schema` on python side whenever it makes sense.

Reply:
so it can not support selecting `struct.*` now?

Reply:
Yes not yet. There is on TODO list. We need proto change accordingly to support it.

Right now we don't even have a complete Struct support yet :)